config/default/traefik.yaml

```diff
@@ -0,0 +1,25 @@
+globalArguments:
+  - "--global.checknewversion"
```

Comment:
What do you think about disabling all the default values from https://github.com/traefik/traefik-helm-chart/blob/6b4c3642141087cc8c92a201a3d8b7ccdec9c1d5/traefik/values.yaml#L177?

* You already removed the statistics 👍 
* Checking for a new version should be the role of `updatecli`. Hence avoid polluting the ingress pod's logs each time a new version is released.

```suggestion
globalArguments: []
```

Reply:
Good point